textDisplay.cc

```diff
@@ -0,0 +1,82 @@
+#include "textdisplay.h"
+
+TextDisplay::TextDisplay(int n):View(n){
+	// Creates a new nxn textdisplay and sets the default value to '0'
+	theDisplay = new char*[n];
+	for (int row = 0; row < n; row++){
```

Comment:
Nice way to assign all the characters! I wonder if we should assign them all " " and "_" and when we load the pieces, then we notify the view that it is changing?